addons/l10n_mx/migrations/2.2/end-migrate.py

```diff
@@ -4,4 +4,4 @@
 def migrate(cr, version):
     env = api.Environment(cr, SUPERUSER_ID, {})
     for company in env['res.company'].search([('chart_template', '=', 'mx')], order="parent_path"):
-        env['account.chart.template'].try_loading('mx', company)
+        env['account.chart.template'].try_loading('mx', company, force_create=False)
```

Comment:
This script for example was [created](https://github.com/odoo/odoo/commit/9fdd8c6e2d3b8f3e41ed9b93d73a1da9e01b12e1) for the purpose of updating taxes, adding force_create=False would skip the fix and make it impossible for dbs to get the new tax

Reply:
There is a different mechanism for updating the CoA using `try_loading`.

It doesn't make sense to rely on the upgrade of Odoo (or its modules) to be compliant with the law.